### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Keyboard: MAS2.1.1: "Time and CPU Time" tool-tip popup of Time Profiler and  "Time and Working set" tool-tip popup of Allocation Profiling are not accessible using Keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#443]
### Prerequisites

1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to ""Start Profiling"" option and invoke it.
5. This will invokes ""Xamarin Profiler"" application.

### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available  instruments to "Time Profiler" and invoke next button. This will invokes Common options window.
3. Navigate to "Start Profiling" button and invoke it. This will launch Time Profiler profiling page. 
4. Try to Invoke "Time and CPU Time" tool-tip popup present on profiler progress table using keyboard(Please refer screenshot).

Scenario 2:

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations" and invoke next button. This will invokes Common options window.
3. Navigate to "Start Profiling" button and invoke it. This will launch Allocations profiling page. 
4. Try to Invoke "Time and Working set" popup present on profiler progress table using keyboard(Please refer screenshot).

### Expected Behavior

Should be able to invoke the "Time and CPU Time" and  "Time and Working set" tool-tip popups using Keyboard.

### Actual Behavior

Scenario 1:
Cannot able to invoke the "Time and CPU Time" tool-tip popup using keyboard.

Note: On hover the mouse over the progress bar "Time and CPU Time" popup is visible.

Scenario 2:
Cannot able to invoke the "Time and Working set" popup using keyboard.

Note: On hover the mouse over the progress bar "Time and CPU Time" popup is visible.

### User Impact

User who rely on keyboard will not be able to access the popup.

### Screenshots

![9 time and cpu](https://user-images.githubusercontent.com/22956130/41720098-9b21fc58-757f-11e8-9617-c990f4eeae8c.png)
![10 time and working set](https://user-images.githubusercontent.com/22956130/41720103-9d7f0180-757f-11e8-9b6a-d7064d8a70b6.png)

### Environment

macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS2.1.1
Xamarin-Profiler-MAC
Wipro
